Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -90,7 +90,22 @@ def uninstall_early_script(directives, **options)
       # :launchctl must come before :quit/:signal for cases where app would instantly re-launch
       def uninstall_launchctl(*services, command: nil, **_)
         booleans = [false, true]
+
+        all_services = []
+
+        # if launchctl item contains a wildcard, find matching process(es)
         services.each do |service|
+          all_services.push(service)
+          next unless /\*/.match?(service)
+
+          find_launchctl_with_wildcard(service)
+            .map { |_, _, id| id }
+            .each do |match|
+            all_services.push(match)
```

Comment:
```suggestion
          all_services += find_launchctl_with_wildcard(service)
```

feels like this would be a nicer API for `find_launchctl_with_wildcard`